Changelog.md

```diff
@@ -16,6 +16,7 @@
 - Added members parameter to add_group_api to support explicit member assignment during group creation (#7481)
 - Modified add_group_api to use username as group name for individual assignments (#7481)
 - Improved styling of markdown previews in `MarkdownPreview` component and annotation displays (#7487)
+- Collect single submission api feature (#7494)
```

Comment:
@donny-wong Oh I forgot to comment on this: please make this changelog entry more descriptive. "Feature" is a generic word and should be avoided. "Added" is a good verb to start with